core/src/main/scala/org/apache/gearpump/cluster/master/MasterSummary.scala

```diff
@@ -23,7 +23,14 @@ object MasterStatus {
   val UnSynced = "unsynced"
 }
 
-case class MasterDescription(leader: (String, Int), cluster: List[(String, Int)], aliveFor: Long,
-                             logFile: String, jarStore: String,
-                             masterStatus: MasterStatus.Type,
-                             homeDirectory: String)
+case class MasterSummary(
+  leader: (String, Int),
+  cluster: List[(String, Int)],
+  aliveFor: Long,
+  logFile: String,
+  jarStore: String,
+  masterStatus: MasterStatus.Type,
+  homeDirectory: String,
+  activities: List[MasterActivity])
+
+case class MasterActivity(time: Long, event: String)
```

Comment:
Do we want the *Summary.scala files copied into services/shared/... so that services/{jvm,js} can access them from the shared directory?